### PR TITLE
[cli] Show private keys for constructed accounts when listing

### PR DESCRIPTION
**Pull request by metajack**
When working with automatically created accounts for testing, it is useful to be able to inspect the private keys. This PR adds them to the output when accounts are listed.

For example, these are useful if you are working with the API and need to construct accounts for testing that will sign something outside the CLI.

### CODE REVIEW

**Review by davidiw — APPROVED** [review]
it is for testing after all....

File: testsuite/cli/libra-wallet/src/key_factory.rs [hunk 1]
```diff
@@ -104,6 +104,11 @@ impl ExtendedPrivKey {
         libra_types::account_address::from_public_key(&self.get_public())
     }
 
+    /// Get private key
+    pub fn get_private_key(&self) -> Ed25519PrivateKey {
+        self.private_key.clone()
```
Comment (davidiw) [inline testsuite/cli/libra-wallet/src/key_factory.rs:109]: looks like you'll need to return a reference or perhaps you want to relabel this to something more insidious such as get_private_key_value and convert it into bytes

Comment (metajack) [reply testsuite/cli/libra-wallet/src/key_factory.rs:109]: I just enabled `features = ["fuzzing"]` for libra-crypto.

**Pull request closed without merging**